geoutils/georaster.py

```diff
@@ -1218,23 +1220,25 @@ def outside_image(self, xi,yj, index=True):
         :returns is_outside: True if ij is outside of the image.
         """
         if not index:
-            xi,xj = self.xy2ij(xi,yj)
+            xi,yj = self.xy2ij(xi,yj)
```

Comment:
It's strange to have the double name x and i or y and j. It's a bit redundant...

Reply:
This is because input can be coordinate or index